src/NuGet.Clients/NuGet.Tools/NuGet.Tools.csproj

```diff
@@ -277,6 +177,132 @@
+      <Project>{06662133-1292-4918-90f3-36c930c0b16f}</Project>
+      <Name>NuGet.SolutionRestoreManager</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\NuGet.VisualStudio.Implementation\NuGet.VisualStudio.Implementation.csproj">
+      <Project>{9623cf30-192c-4864-b419-29649169ae30}</Project>
+      <Name>NuGet.VisualStudio.Implementation</Name>
+      <Private>True</Private>
+    </ProjectReference>
+    <ProjectReference Include="..\Options\NuGet.Options.csproj">
+      <Project>{53ab7187-cdf3-48b5-b90e-8919d3cb904d}</Project>
+      <Name>NuGet.Options</Name>
+      <Private>True</Private>
+    </ProjectReference>
+    <ProjectReference Include="..\PackageManagement.PowerShellCmdlets\NuGet.PackageManagement.PowerShellCmdlets.csproj">
+      <Project>{26dc17ac-a390-4515-a2c0-07a0950036c5}</Project>
+      <Name>NuGet.PackageManagement.PowerShellCmdlets</Name>
+      <Private>True</Private>
+    </ProjectReference>
+    <ProjectReference Include="..\PackageManagement.UI\NuGet.PackageManagement.UI.csproj">
+      <Project>{538adefd-2170-40a9-a2c5-ec8369cfe490}</Project>
+      <Name>NuGet.PackageManagement.UI</Name>
+      <Private>True</Private>
+    </ProjectReference>
+    <ProjectReference Include="..\PackageManagement.VisualStudio\NuGet.PackageManagement.VisualStudio.csproj">
+      <Project>{306cddfa-ff0b-4299-930c-9ec6c9308160}</Project>
+      <Name>NuGet.PackageManagement.VisualStudio</Name>
+      <Private>True</Private>
+    </ProjectReference>
+    <ProjectReference Include="..\TeamFoundationServer\NuGet.TeamFoundationServer.csproj">
+      <Project>{e961d080-f10b-4f11-bea0-056509c086cc}</Project>
+      <Name>NuGet.TeamFoundationServer</Name>
+      <Private>True</Private>
+    </ProjectReference>
+    <ProjectReference Include="..\NuGet.VisualStudio.Interop\NuGet.VisualStudio.Interop.csproj">
+      <Project>{7DB43FE1-75E1-49F9-B2C8-06A552BA2144}</Project>
+      <Name>NuGet.VisualStudio.Interop</Name>
+      <Private>True</Private>
+    </ProjectReference>
+    <ProjectReference Include="..\VisualStudio.Facade\NuGet.VisualStudio.Facade.csproj">
+      <Project>{eea49a74-6efc-410e-9745-bad367ac151d}</Project>
+      <Name>NuGet.VisualStudio.Facade</Name>
+      <Private>True</Private>
+    </ProjectReference>
+    <ProjectReference Include="..\NuGet.VisualStudio\NuGet.VisualStudio.csproj">
+      <Project>{e5556bc6-a7fd-4d8e-8a7d-7648df1d7471}</Project>
+      <Name>NuGet.VisualStudio</Name>
+      <Private>True</Private>
+    </ProjectReference>
+    <ProjectReference Include="..\VsConsole\Console\NuGet.Console.csproj">
+      <Project>{50e33da2-af14-486d-81b8-bd8409744a38}</Project>
+      <Name>NuGet.Console</Name>
+      <Private>True</Private>
+    </ProjectReference>
+    <ProjectReference Include="..\VsConsole\PowerShellHost\NuGetConsole.Host.PowerShell.csproj">
+      <Project>{5a79eef3-51c0-4a14-8d37-50ef38ad835d}</Project>
+      <Name>NuGetConsole.Host.PowerShell</Name>
+      <Private>True</Private>
```

Comment:
It's just how it was before. 
I have removed them now, since I don't see the benefit from it now.